.changeset/tall-parents-deny.md

```diff
@@ -0,0 +1,21 @@
+---
+'@backstage/plugin-search-react': minor
+---
+
+`SearchContextProviderForStorybook` and `SearchApiProviderForStorybook` has been deleted. New mock implementation of the `SearchApi` introduced. If you need to mock the api we recommend you to do the following:
+
+```tsx
+import {
+  searchApiRef,
+  MockSearchApi,
+  SearchContextProvider,
+} from '@backstage/plugin-search-react';
+import { ApiProvider } from '@backstage/core-app-api';
+import { TestApiRegistry } from '@backstage/test-utils';
+
+<ApiProvider apis={TestApiRegistry.from([searchApiRef, new MockSearchApi()])}>
```

Comment:
Realized this is the most important place for ^ comment, let's recommend the simplest method

Reply:
See https://github.com/backstage/backstage/pull/11164/commits/79115aeb5c16999875261974cd4f76f1d20af8a4